VOTable.tex

```diff
@@ -1140,14 +1140,14 @@ \subsection{Units}
 some physical unit,
 which is specified by the {\attr{unit}}
 attribute of the {\elem{FIELD}}.
-The  syntax of the {\em unit} string is defined in
-the VOUnit specification, \citet{2014ivoa.spec.0523D};
-it is basically written as a string without blanks or spaces,
-where the symbols {\bf.} or {\bf*} indicate a multiplication,
-{\bf/} stands for the division, and no special symbol is required
-for a power.
-Examples are \attrval{unit}{m2} for m$^2$,
-\attrval{unit}{cm-2.s-1.keV-1} for cm$^{-2}$s$^{-1}$keV$^{-1}$,
+The syntax of the {\em unit} string SHOULD conform to
+the VOUnits specification, \citet{2014ivoa.spec.0523D};
+this requires a string without blanks or spaces
+where multiplication is indicated by the symbol ``{\tt.}'',
+division by the symbol ``{\tt/}''
+and exponentiation by the symbol ``{\tt**}''.
+Examples are \attrval{unit}{m**2} for m$^2$,
+\attrval{unit}{cm**-2.s**-1.keV**-1} for cm$^{-2}$s$^{-1}$keV$^{-1}$,
 or \attrval{unit}{erg/s} for erg\,s$^{-1}$.
```

Comment:
I know this `erg/s` value didn't change with this PR, but I notice that `erg` is considered deprecated according to VOUnit 1.0 with the same language in the proposed [VOUnit 1.1](https://www.ivoa.net/documents/VOUnits/20230502/PR-VOUnits-1.1-20230502.html#tth_sEc2.8).  

Should we take this opportunity to change the example to not use `erg`?  Something like:
```suggestion
or \attrval{unit}{10**-7.J/s} for erg\,s$^{-1}$
(uses \literalvalue{10**-7.J} in place of the deprecated \literalvalue{erg}).
```

Reply:
Good point, though given that the text here is only intended to be a very brief introduction to the syntax it might be better to sidestep the status of erg by just using something like `m/s` instead or omit the final example altogether.

Reply:
I like the idea of simplifying or removing the last example to avoid `erg`.  Its presence was not very important for this section.